x-pack/plugins/cloud_security_posture/server/plugin.test.ts

```diff
@@ -82,6 +83,7 @@ describe('Cloud Security Posture Plugin', () => {
       taskManager: taskManagerMock.createStart(),
       security: securityMock.createStart(),
       licensing: licensingMock.createStart(),
+      dataViews: createIndexPatternsStartMock(),
```

Comment:
I added the Dataviews server mock to fix the type checker

Reply:
👑